app/views/staff/events/show.html.haml

```diff
@@ -1,158 +1,227 @@
 .event
   .row
-    .col-md-12
-      .page-header
-        %h1= event
+    .col-md-12.page-header.info-bar
```

Comment:
Is there any reason you moved the page-header.info-bar up to be part of the .col-md-12 div? 

I would slightly prefer: 
.col-md-12
      .page-header.info-bar

Just curious, though.